app/src/main/res/values/strings.xml

```diff
@@ -241,52 +241,67 @@
   <string name="nominated_see_more"><u>See webpage for details</u></string>
   <string name="view_browser">View in Browser</string>
 
-    <string name="nearby_location_has_not_changed">Location has not changed.</string>
-    <string name="nearby_location_not_available">Location not available.</string>
-    <string name="location_permission_rationale_nearby">Permission required to display a list of nearby places</string>
-    <string name="get_directions">GET DIRECTIONS</string>
-    <string name="read_article">READ ARTICLE</string>
+  <string name="nearby_location_has_not_changed">Location has not changed.</string>
```

Comment:
Aah! This needs to be fixed Tanvi. :)

Formatting issues are still happening whenever you modify existing files. If you need help with this, let me or @misaochan know.